packages/flame/lib/src/components/shape_component.dart

```diff
@@ -12,7 +12,7 @@ class ShapeComponent extends PositionComponent {
   ShapeComponent(
     this.shape,
     this.shapePaint, {
-    Anchor anchor = Anchor.center,
+    Anchor anchor = Anchor.topLeft,
```

Comment:
:thinking:

Reply:
Since all of our other components are `topLeft` by default I thought that it would be sensible to have it here too.
I think I put a comment regarding that on `shape.toComponent`.